CHANGELOG.md

```diff
@@ -38,6 +38,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - `SamplingResult.TraceState` is correctly propagated to a newly created
   span's `SpanContext`. (#1655)
 - Jaeger Exporter: Ensure mapping between OTEL and Jaeger span data complies with the specification. (#1626)
+- Issue #1490, apply same logic as in the SDK. (#1687)
```

Comment:
```suggestion
- Reduced excess memory usage by global `TracerProvider`. (#1687)
```

We should have this describe the thing that was fixed rather than simply referring to an issue.